src/content/community/contributor-roles/_index.en.md

```diff
@@ -179,6 +185,23 @@ The following apply to people who would be an owner:
   * Identifying subtle or complex issues in designs and implementation PRs
 * Directly contributed to the project through implementation and / or review
 
+### Owner Removal and Future Elected Governance
+
+Removal of Project Owners is currently frozen except for stepping down or violations of the Code of Conduct. This is a temporary governance
+step to define a removal process for extreme cases while protecting the project from dominance by a company. Once the Submariner community
+is diverse enough to replace Project Owners with an elected governance system, the project should do so. If the project hasn't replaced
+Project Owners with elected governance in two years, and if there are committers from at least three different companies, the project
```

Comment:
This is like a sign saying “back in five minutes” on a shop. If we’re going to specify a calendar, we need to have actual dates here.

Reply:
Maybe one day we'll be able to run `git blame` against physical signs, but yes I see the point and made it explicit. :wink: :pray: